Objects/typeobject.c

```diff
@@ -9233,7 +9515,7 @@ slot_bf_getbuffer(PyObject *self, Py_buffer *buffer, int flags)
 }
 
 static int
-releasebuffer_maybe_call_super(PyObject *self, Py_buffer *buffer)
+releasebuffer_maybe_call_super_unlocked(PyObject *self, Py_buffer *buffer, releasebufferproc *base_releasebuffer)
```

Comment:
I think this would be easier to understand if it returned a `releasebufferproc` (possibly NULL) instead of coordinating the error return code with the pointer write.

The `-1` error return doesn't do anything because we never set a Python error.

Reply:
Yeah weird, it doesn't look like it was discussed in the original PR.  I'll just get rid of the unused error handling stuff.